akka-remote/src/main/scala/akka/remote/artery/AeronSource.scala

```diff
@@ -92,7 +93,7 @@ private[remote] class AeronSource(
 
       private val sub = aeron.addSubscription(channel, streamId)
       // spin between 100 to 10000 depending on idleCpuLevel
-      private val spinning = 1100 * taskRunner.idleCpuLevel - 1000
+      private val spinning = if (inboundSourceSpinning < 0) 1100 * taskRunner.idleCpuLevel - 1000 else inboundSourceSpinning
```

Comment:
We should derive it from the cpu-level and the inbound-lanes setting. We can calculate the value in ArteryTransport and pass in the parameter to the stage.

My suggestion is that we change it to:
inbound-lanes > 1 => spinning = 0

inbound-lanes == 1 =>
  cpu-level < 5 => spinning = 0
  else => something like `spinning = 50 * idleCpuLevel - 240`


The reason I want to keep the spinning is that I see a drop in the throughput when spinning=0 with inbound-lanes=1